tasks/main.yml

```diff
@@ -6,7 +6,7 @@
         - '{{ ansible_distribution }}-{{ ansible_distribution_release }}.yml'
         - '{{ ansible_distribution }}.yml'
         - '{{ ansible_os_family }}.yml'
-      skip: true
+      errors: "ignore"
```

Comment:
despite the warning this change doesn't work with ansible 2.8. please revert

Reply:
ok. done.